### PR TITLE
feat: Add Shareville logo

### PR DESCRIPTION
**Pull request by JCB-K**
Add 2 version of the Shareville logo: with and without logotype

### CODE REVIEW

**Comment by iamstarkov** [comment]
I would suggest to make it separate `SharevilleLogo` component

**Comment by iamstarkov** [comment]
i mean without changin existing `Logo`

**Comment by bstream** [comment]
I agree with @iamstarkov . The line 
```
All props except for `height` and `onlyIcon` are ignored.
```
in particular indicates that it should be handled differently and therefore it makes sense to have it in a separate component.

**Comment by JCB-K** [comment]
@bstream @iamstarkov done

File: src/components/shareville-logo/index.jsx [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+function SharevilleLogo({ onlyIcon, height }) {
+  const width = onlyIcon ? height : height * 4.89;
+
+  if (onlyIcon) {
+    return (
+      <svg viewBox="0 0 166.54 203.5" width={width} height={height}>
+        <defs>
+          <style
+            dangerouslySetInnerHTML={{
+              __html: '.svLogoGradientClass-1{fill:url(#linear-gradient);}.svLogoGradientClass-2{fill:url(#linear-gradient-2);}.svLogoGradientClass-3{fill:url(#linear-gradient-3);}',
```
Comment (codler) [inline src/components/shareville-logo/index.jsx]: Why `svLogoGradientClass` as name? Is it autogenerated? Would be better if it contain shareville like the class below

Comment (JCB-K) [reply src/components/shareville-logo/index.jsx]: @codler Fixed this now in a new release. (It's generated and never manually used by devs so not a big deal)

Comment (codler) [reply src/components/shareville-logo/index.jsx]: no but next time someone generate svg it might get same name and will then conflict.

Comment (JCB-K) [reply src/components/shareville-logo/index.jsx]: @codler ah I see what you mean. I already customised it by prefixing it with `svLogo`. (by default it was `gradientclass-1`, which is very generic). But good cal!

**Pull request merged**